.gitignore

```diff
@@ -4,5 +4,7 @@ tmp/
 .vs/
 .idea/
 
+proto/googleapis
+
```

Comment:
(Adding here as I can't add a comment on the submodule add itself...)

It's not clear to me yet (and hopefully it'll *become* clear) why we've got the submodule at all. The intention of `third_party/googleapis` was to capture enough of `googleapis` for the events, without inadvertently allowing undesirable dependencies.

If we *do* want it, I think it would be better in `third_party` or some other directory separated from `proto` - having `proto/google` and `proto/googleapis/google` as the logically-equivalent proto paths is odd.

Reply:
There should be no cloning of googleapis, this was never used as part of the `protoc` generation process and was a mistake adding it. This instruction is outdated.